src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -122,8 +123,12 @@ public RubyContext(RubyLanguage language, TruffleLanguage.Env env) {
                     "this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
         }
 
-        rubyHome = findRubyHome();
-        Log.LOGGER.config(() -> String.format("ruby home: %s", rubyHome));
+        try {
+            rubyHome = findRubyHome();
+        } catch (IOException e) {
+            throw new JavaException(e);
+        }
+        Log.config("ruby home: %s", rubyHome);
```

Comment:
I don't see any difference:
```
$ jt ruby -Xlog=config -e0
[ruby] CONFIG ruby home: /home/eregon/code/truffleruby-ws/truffleruby
```

Besides, all other logging is done via the `Log` class.
My motivation is avoiding a lambda here during startup, for something that really does not need it.